### DISCUSSION TITLE
"why use carbon?" is not clear enough ,like every other lang.(except rust , c[++])

### DISCUSSION THREAD

**Discussion by gintominto5329** [carbon-language/carbon-lang#3608] [General]
thanks

**Comment by chandlerc** [comment]
I'm sorry that the documentation we have isn't clear enough, but without specific suggestions about how the documentation could be improved and made better, I'm not sure there is anything to do here.

We also work hard to not spend our time discussing problems with other languages. If you'd like to suggest improvements to other languages' documentation, that's best done in their forums or issues.

I'm closing the thread because of the negativity towards onther languages, and as this doesn't seem to be a topic of discussion, but just mentioning a complaint. If there is a suggestion for how we could improve the documentation, please file an issue with a clear suggestion on what needs to be improved, and even better if you have ideas on how to improve it.
